packages/react-reconciler/src/__tests__/ReactFiberEvents-test-internal.js

```diff
@@ -5,7 +5,6 @@
  * LICENSE file in the root directory of this source tree.
  *
  * @emails react-core
- * @jest-environment node
```

Comment:
This existed from a copy and paste from a previous test file. We don't want it here as we want Jest to give the JSDOM for `document` and using this setting removes JSDOM support for this test.